docs/aks/README.md

```diff
@@ -65,6 +65,22 @@ az account list -o table
 # select the Azure account
 az account set -s {subscription name or Id}
 
+# save the subscriptionID as a variable
+export He_Sub=$(az account show --query id -o tsv)
+```
+
+Save your environment variables for ease of reuse and picking up where you left off.
+
+```bash
+
+# run the saveenv.sh script at any time to save He_* variables to ~/${He_Name}.env
+# make sure you are in the root of the repo
+cd $REPO_ROOT
+./saveenv.sh
+
+# at any point if your terminal environment gets cleared, you can source the file
+# you only need to remember the name of the env file (or set the $He_Name variable again)
+source ~/{yoursameuniquename}.env
```

Comment:
Not necessarily for this PR, but I'm wondering if we can't standardize on a name to support copy / run?  Since it's in "your" home directory, we shouldn't have to worry about conflicts and this is [one of] the only place where you can't copy / run.

Reply:
I will look into it as that saveenv.sh is in a bunch of our other run throughs. I will discuss with @atxryan who created it to see if we can't make it more elegant.